datadog_lambda/handler.py

```diff
@@ -27,5 +51,11 @@ class HandlerError(Exception):
 
 (mod_name, handler_name) = parts
 modified_mod_name = modify_module_name(mod_name)
-handler_module = import_module(modified_mod_name)
-handler = datadog_lambda_wrapper(getattr(handler_module, handler_name))
+
+try:
+    start_time_ns = time.time_ns()
+    handler_module = import_module(modified_mod_name)
+    handler_func = getattr(handler_module, handler_name)
+    handler = datadog_lambda_wrapper(handler_func)
+except Exception as e:
+    handler = _ErrorFallbackHandler(e, modified_mod_name, start_time_ns)
```

Comment:
I just want to call out that this changes the behavior of the function slightly. Now instead of failing during the init phase, it will fail during the execution phase. Totally not a big deal, but good to point out that customers aren't charged for init duration, only for duration duration. This means whereas before they were not charged for these failures, now they will be.

Reply:
Ah I see, that makes sense. Thanks for pointing that out!